packages/functions/src/aggregator/handle-inclusion-insert-to-update-state.js

```diff
@@ -0,0 +1,94 @@
+import * as Sentry from '@sentry/serverless'
+import { Table } from 'sst/node/table'
+import { unmarshall } from '@aws-sdk/util-dynamodb'
+
+import { decodeRecord } from '@w3filecoin/core/src/store/aggregator-inclusion-store.js'
+import { createClient as createPieceStoreClient } from '@w3filecoin/core/src/store/aggregator-piece-store.js'
+import * as aggregatorEvents from '@web3-storage/filecoin-api/aggregator/events'
+
+import { mustGetEnv } from '../utils.js'
+
+Sentry.AWSLambda.init({
+  environment: process.env.SST_STAGE,
+  dsn: process.env.SENTRY_DSN,
+  tracesSampleRate: 1.0,
+})
+
+/**
+ * @typedef {import('@w3filecoin/core/src/store/types').AggregatorInclusionRecord} AggregatorInclusionRecord
+ * @typedef {import('@w3filecoin/core/src/store/types').AggregatorInclusionStoreRecord} AggregatorInclusionStoreRecord
+ */
+
+/**
+ * Get EventRecord from the SQS Event triggering the handler.
+ * On Inclusion store insert, piece table can be updated to reflect piece state.
+ *
+ * @param {import('aws-lambda').DynamoDBStreamEvent} event
+ */
+async function handleInclusionInsertToUpdateState (event) {
+  // Get context
+  const context = getContext()
+
+  // Parse records
+  const eventRawRecords = parseDynamoDbEvent(event)
+  // if one we should put back in queue
+  if (eventRawRecords.length !== 1) {
+    return {
+      statusCode: 400,
+      body: `Expected 1 DynamoDBStreamEvent per invocation but received ${eventRawRecords.length}`
+    }
+  }
+  /** @type {AggregatorInclusionStoreRecord} */
+  // @ts-expect-error can't figure out type of new
+  const storeRecord = unmarshall(eventRawRecords[0].new)
+  const record = decodeRecord(storeRecord)
+
+  const { ok, error } = await aggregatorEvents.handleInclusionInsertToUpdateState(context, record)
+  if (error) {
+    return {
+      statusCode: 500,
+      body: error.message || 'failed to handle inclusion insert event'
+    }
+  }
```

Comment:
Not throwing here will remove the stack from the logs, so while you get a nice message in the body, it might be harder to debug.

I'm using this helper (Irakli suggested) to make throwing a little easier:

https://github.com/web3-storage/w3infra/blob/8bdd528a284446aed93db024ee1323aabc122dbf/billing/functions/lib.js#L11-L22

The code above becomes:

```suggestion
  const ok = expect(
    await aggregatorEvents.handleInclusionInsertToUpdateState(context, record),
    'failed to handle inclusion insert event'
  )
```

Reply:
Cool, I will see how to adopt this.
One thing I would like, is to rely on status codes to infer metrics and this would make it difficult unless we explore this a bit further. Will leave in scope of this PR